pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -163,6 +188,15 @@ func getWPAClient(timeout time.Duration) (wpa_client.Interface, error) {
 	return wpa_client.NewForConfig(clientConfig)
 }
 
+func getKubeDynamicClient(timeout time.Duration) (dynamic.Interface, error) {
```

Comment:
there is a function `getDynamicKubeClient` just above, that does the same thing :D